sheets/sort

```diff
@@ -17,7 +20,7 @@ sort -r /usr/share/dict/british-english
 printf '%s\n' this is a list of of random words with duplicate words | sort -u
 
 # Sort numerically. If you don't provide the `-n` flag, sort(1) will instead
-# sort by the ASCI table, as mentioned above, meaning it'll display as 1, 10, -
+# sort by the ASCII table, as mentioned above, meaning it'll display as 1, 10, -
```

Comment:
That's me you're correcting, and I'm happy for it! Lol For the longest time, I got confused between ASCII and ANSI.